feathr_project/feathr/definition/sink.py

```diff
@@ -1,12 +1,18 @@
-from typing import List, Optional
+from abc import abstractmethod
+import json
+from typing import Dict, List, Optional
 from jinja2 import Template
+from pymysql import paramstyle
```

Comment:
Remove pymysql as current code does not use it

Reply:
Removed